deployments/setup-fork.ts

```diff
@@ -41,6 +42,14 @@ const fundAccount = async (account: string, fundingRequests: FundingRequest[]) =
     Logger.log(`Funding ${account}...`);
 
     for (const fundingRequest of fundingRequests) {
+        // for tokens which are missing skip funding request
+        if (fundingRequest.token === ZERO_ADDRESS) {
+            continue;
+        }
+        const { whale } = fundingRequest;
```

Comment:
Super nitpick: you can use `whale` below as well as fetching `token` and use it directly as well (instead of `fundingRequest.token`)

Reply:
fixed